web/src/components/DLP/OverviewTab.vue

```diff
@@ -67,6 +67,30 @@
       </template>
     </MetadataCard>
 
+    <MetadataCard
+      :items="protocols"
+      v-if="protocols && protocols.length"
+      name="Protocols"
+      icon="mdi-protocol"
```

Comment:
```suggestion
      icon="mdi-rodent"
```

Reply:
indeed `://` is quite cryptic... but I am not sure if a mouse (i.e.)
would be not  misleading -- since it might be not even about mice research and/or could be mis-treated as "species" or some other aspect.  May be [file-document-check](https://pictogrammers.com/library/mdi/icon/file-document-check/) as "protocol" being a checklist to follow?

![image](https://github.com/user-attachments/assets/8f7b9725-4c18-46d7-8d40-79826d789929)

```suggestion
      icon="mdi-file-document-check"
```

?

Reply:
I like it.